bootstrap/services/druid-bundle.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "DRUID",
+  "registerClass": "com.hortonworks.streamline.streams.cluster.register.impl.DruidServiceRegisterer",
+  "serviceUISpecification": {
+    "fields": [
+      {
+        "uiName": "common.runtime.properties",
```

Comment:
OK. Other uiNames for configuration files are similar, so I'll make the change for all places.